src/components/ImageDetails/ReuseSurvey.vue

```diff
@@ -1,24 +1,14 @@
 <template>
```

Comment:
We could delete this file as part of this PR, since the form no longer exists and it isn't in use.

```
front on  main +/- [$?⇣] via  v16.13.0
❯ rg 'ReuseSurvey'
src/components/ImageDetails/ReuseSurvey.vue
6:      @click="onReuseSurveyClick"
7:      @keyup.enter="onReuseSurveyClick"
29:  name: 'ReuseSurvey',
45:    onReuseSurveyClick() {

test/unit/specs/components/ImageDetails/reuse-survey.spec.js
9:import ReuseSurvey from '~/components/ImageDetails/ReuseSurvey'
```